src/main/java/com/jcabi/aether/Aether.java

```diff
@@ -95,6 +95,9 @@
  *  and verify the expected HTTP requests have been sent to the mocked server.
  * @todo #69:30min Fix NullPointerExceptions in CL build for Open JDK6.
  * @todo #69:30min Fix UnsupportedClassVersionError in CL build for Open JDK6.
+ * @todo #82:30min Clean unnecessary docker setup (src/docler/Dockerfile) and set
+ *  versions to the used packages (e.g. lock java version instead of default
+ *  jdk). Discuss with architect.
```

Comment:
@amihaiemil I don't think that much discussion is needed here. I would stress that the goal here is to have minimal docker image with all versions locked that is enough to build this project.